app/partials/app.cjsx

```diff
@@ -3,15 +3,81 @@ auth = require 'panoptes-client/lib/auth'
 IOStatus = require './io-status'
 MainHeader = require './main-header'
 MainFooter = require './main-footer'
+GeordiClient = require 'zooniverse-geordi-client'
 {generateSessionID} = require '../lib/session'
 
+class GeordiLogger
+  constructor: (@state, @geordi) ->
+
+  keys: {}
+
+  geordi: null
+
+  instance: =>
+    @geordi = @geordi || @makeGeordi @keys?.projectToken
+
+  makeGeordi: (projectSlug) ->
+    new GeordiClient
+      server: @state?.env
+      projectToken: projectSlug || ''
```

Comment:
Hmm actually I see it is captured here https://github.com/zooniverse/Panoptes-Front-End/pull/2590/files#diff-83addd0f5535ee9ffe878b365f61712bR89 - where is it being stored in Geordi?  (sorry i think I am just missing this!)